src/event.rs

```diff
@@ -9,14 +9,11 @@ pub enum Event<'a> {
     StartTag {
         name: Cow<'a, str>,
         attrs: AttributeSet<'a>,
+        is_self_closing: bool,
```

Comment:
I'm not entirely sure, but I think it is ok for member _fields_ (which are flags) to not have `is_` prefix (it's needed for methods (which are checks), I guess).